pkgs/by-name/sw/swhkd/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+}:
+let
+  name = "swhkd";
+  version = "30f25b5bf99df5f16d91b51a7bd397c1de075085";
+  buildInputs = with pkgs; [ systemd ];
+  nativeBuildInputs = with pkgs; [ pkg-config ];
+in
+rustPlatform.buildRustPackage rec {
+  inherit name buildInputs nativeBuildInputs;
+  src = fetchFromGitHub {
+    owner = "waycrate";
+    repo = name;
+    rev = version;
+    hash = "sha256-K05v2xw7G/ZJsf62g696TOSZHMAOWf24sT9JcvvBNx0=";
+  };
+  cargoHash = "sha256-E5AE18CfeX1HI/FbGDFoUDsPyG/CpJrD+8Ky7c+EQUw=";
+  meta = {
+    description = "Sxhkd clone for Wayland (works on TTY and X11 too)";
+    homepage = "https://github.com/waycrate/swhkd";
+    license = lib.licenses.bsd2;
```

Comment:
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.